5calls/app/src/main/res/values/strings.xml

```diff
@@ -365,4 +368,13 @@
     <!-- Action for a snackbar or button which brings you to the location update screen [CHAR_LIMIT=10] -->
     <string name="update">update</string>
 
+    <!-- URL to sign up for the 5 Calls newsletter [CHAR_LIMIT=NONE] -->
+    <string name="newsletter_url" translatable="false">https://my.sendinblue.com/users/subscribe/js_id/2p22o/id/1</string>
```

Comment:
Yeah, that's a great idea @dektar. Let's assume that url and I'll add a redirect to the backend shortly.